### DISCUSSION TITLE
Has the `SnippetTextEdit` object and some api using it been removed?

### DISCUSSION THREAD

**Discussion by ArturoDent** [microsoft/vscode-discussions#481] [Extension Development QnA] [answered]
I ask because I get no intellisense for `vscode.SnippetTextEdit` and `someWorkspaceEdit.set()` functions that can use `SnippetTextEdit`.  But they are both in the documentation, e.g. https://code.visualstudio.com/api/references/vscode-api#SnippetTextEdit.  Curious...

**Comment by isidorn** [comment, marked as answer]
I get the intellisense for `vscode.SnippetTextEdit`. Do you still see this issue with the latest VS Code Insiders?

**Comment by ArturoDent** [comment]
I can't replicate it anymore in Stable or Insiders with the current builds.  Thanks for checking.
